lib/takeaway.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'messager'
+
+class Takeaway
+
+  attr_reader :menu, :basket
+
+
+  def initialize (messager)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)